management/dbaas/psmdb_cluster_test.go

```diff
@@ -0,0 +1,207 @@
+package dbaas
+
+import (
+	"testing"
+
+	dbaasClient "github.com/percona/pmm/api/managementpb/dbaas/json/client"
+	psmdbcluster "github.com/percona/pmm/api/managementpb/dbaas/json/client/psmdb_cluster"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/codes"
+
+	pmmapitests "github.com/Percona-Lab/pmm-api-tests"
+)
+
+const (
+	psmdbKubernetesClusterName = "api-test-k8s-mongodb-cluster"
+)
+
+//nolint:funlen
+func TestPSMDBClusterServer(t *testing.T) {
+	if pmmapitests.Kubeconfig == "" {
+		t.Skip("Skip tests of PSMDBClusterServer without kubeconfig")
+	}
+	registerKubernetesCluster(t, psmdbKubernetesClusterName, pmmapitests.Kubeconfig)
+
+	t.Run("BasicPSMDBCluster", func(t *testing.T) {
+		paramsFirstPSMDB := psmdbcluster.CreatePSMDBClusterParams{
+			Context: pmmapitests.Context,
+			Body: psmdbcluster.CreatePSMDBClusterBody{
+				KubernetesClusterName: psmdbKubernetesClusterName,
+				Name:                  "first-psmdb-test",
+				Params: &psmdbcluster.CreatePSMDBClusterParamsBodyParams{
+					ClusterSize: 3,
+					Replicaset: &psmdbcluster.CreatePSMDBClusterParamsBodyParamsReplicaset{
+						ComputeResources: &psmdbcluster.CreatePSMDBClusterParamsBodyParamsReplicasetComputeResources{
+							CPUm:        1,
```

Comment:
Any idea what is the minimal requirements for CPUm?

Reply:
Unfortunatelly I don't know :(